carpeta_salida = r"\\ntappama01-c01\Herramienta Usuario Final en Ejecucion\REVISION XPRESS"
if not os.path.exists(carpeta_salida):
    os.makedirs(carpeta_salida, exist_ok=True)
ruta_pdf = os.path.join(carpeta_salida, f"{pdf_name}.pdf")



\\ntappama01-c01\Herramienta Usuario Final en Ejecucion\REVISION XPRESS

carpeta_salida = r"\\ntappama01-c01\Herramienta Usuario Final en Ejecucion\REVISION XPRESS"
if not os.path.exists(carpeta_salida):
    # Opcional: crear carpeta, o lanzar error si no existe o sin permiso
    os.makedirs(carpeta_salida, exist_ok=True)
ruta_pdf = os.path.join(carpeta_salida, f"{pdf_name}.pdf")



Referencia hard.xlsx ignorada por no tener suficientes columnas
Traceback (most recent call last):
  File "C:\Users\katejada\Desktop\Codigos y .Exe\ACH\cross3.py", line 150, in <module>
    raise ValueError("No se pudieron procesar archivos de referencia v치lidos.")
ValueError: No se pudieron procesar archivos de referencia v치lidos.


error 2 
Error procesando referencia hard.xlsx: Unencrypted document
Traceback (most recent call last):
  File "C:\Users\katejada\Desktop\Codigos y .Exe\ACH\cross3.py", line 195, in <module>
    raise ValueError("No se pudieron procesar archivos de referencia v치lidos.")
ValueError: No se pudieron procesar archivos de referencia v치lidos.
